kotest-assertions/src/commonMain/kotlin/io/kotest/properties/Gen.kt

```diff
@@ -44,6 +44,9 @@ interface Gen<T> {
    */
   fun random(seed: Long? = null): Sequence<T>
 
+  /**
+   * @return the [Shrinker] for this gen or `null`
```

Comment:
I believe this is a little bit redundant, isn't it?

Reply:
1. It was the only un-commented function in `Gen` so was thinking it should have something just to appear as fully supported as everything else
1. Overall, it more quickly directs a user to look at `Shrinker` for usage/creation. This method differs IMO from the 2 proceeding methods `constants` and `random` as the return type is a Kotlintest type and not a Kotlin std lib type. Thus it's important for the reader to go understand the `Shinker` type in addition to the semantic method name. Then to avoid duplicating of the `Shrinker` documentation I just linked to it.
1. Also must admit it took me entirely too long to read shrink-er correctly. Thus I may simply have a grudge that redundancy would have helped

Could change the comment to have something like "Honey, I shrunk the `Gen`" as that might have helped with that last point 😆